pubspec.yaml

```diff
@@ -18,3 +18,9 @@ dev_dependencies:
   dependency_validator: '>=2.0.3 <4.0.0'
```

Comment:
We should also update the minimum of dart_style to ^2.0.0 and dependency_validator to ^3.0.0